### ISSUE TITLE
[Spell] Lightwell

### ISSUE THREAD

**Issue by stutte** [dalaranwow/dalaran-wow#2377]
Hello, 

Lightwell bugs upon entering combat, you can only click it once and after that it no longer works. The mouseover doesn't change to an interactable object (even before the first click) either. However, lightwell works perfectly fine when not in combat, you can click it, the mouse changes and it stays clickable untill it's completely depleted of it's charges. So it's only not working properly when in combat, but since lightwell is such an important heal, it's quite annoying when it doesn't work properly. 

Thanks in advance

**Comment by Rushor** [comment]
Scripts/Pet: Lightwell - Move script to TotemAI

* closes https://github.com/dalaranwow/dalaran-wow/issues/2377
* closes 284136fa61ecfe8f04377f0371ac297b99b327ad

The issue was fixed by a developer and will now be tested on the Public Test Realm (PTR). The changes/fixes will be applied with the NEXT changelog on the liverealm.
This means that the issue exists in game at the moment i wrote this message.
Only once the issue was marked with "Fixed - On Live Server", you can find the fix on you server.
Please watch http://www.dalaran-wow.com/news/ to see when the changelog and the fix was applied.

**Comment by AttendantTock** [comment]
Hello,

I have seen two different behaviours:

When the lightwell is placed **while being in combat**: 

1) It looks like the lightwell "goes inactive" right after it's clicked. **It only works once**.
![capture d ecran 2017-10-22 a 23 26 28](https://user-images.githubusercontent.com/29557022/31867070-b213d28c-b780-11e7-9f6e-ebdc299901ec.png)

2) This states perdures even out of combat:
![capture d ecran 2017-10-22 a 23 27 22](https://user-images.githubusercontent.com/29557022/31867071-b22c5546-b780-11e7-9842-b2412344e988.png)
And **it's impossible to click the lightwell again**, in or out of combat.

The other behaviour is when the lightwell is **placed out of combat**:

1) It also "goes inactive", the "interaction" pointer with the golden cog goes away and the regular amor plate hand appears. However **you can click it as normal**.
![capture d ecran 2017-10-22 a 23 29 39](https://user-images.githubusercontent.com/29557022/31867140-e7e24b18-b781-11e7-92a2-0c864abbd9a3.png)

2) You can click it as many times as you want, in or out of combat:
![capture d ecran 2017-10-22 a 23 40 31](https://user-images.githubusercontent.com/29557022/31867190-8ca6c52a-b782-11e7-83ae-955d7fba4c20.png)


In short:
If cast while in combat, lightwell will only work once.
If cast out of combat, it will work normally.
In both cases, the lightwell gets targetable (blue friendly) and the interaction cursor disappear after the first use.
